api/desecapi/tests/base.py

```diff
@@ -1338,8 +1344,9 @@ def _test_rr_sets(cls, subname=None, type_=None, records=None, ttl=None):
     def setUp(self):
         super().setUp()
         # TODO this test does not cover "dyn" / auto delegation domains
-        self.my_empty_domain = self.create_domain(suffix="", owner=self.owner)
-        self.my_rr_set_domain = self.create_domain(suffix="", owner=self.owner)
+        domain_kwargs = {"suffix": self.AUTO_DELEGATION_DOMAINS if self.DYN else None}
+        self.my_empty_domain = self.create_domain(owner=self.owner, **domain_kwargs)
+        self.my_rr_set_domain = self.create_domain(owner=self.owner, **domain_kwargs)
```

Comment:
1. Remove TODO in line above
2. Use `suffix=self.AUTO_DELEGATION_DOMAINS if self.DYN else None` instead of creating a kwargs object with a single entry